client/commonFramework/code-storage.js

```diff
@@ -54,6 +58,7 @@ window.common = (function(global) {
         console.log('unable to save to storage');
         return code;
       }
+      replaceNoprotect(code);
```

Comment:
You'll need to move this inline as strings are immutable this doesn't effect the line below

```js
localStorage.setItem(key + 'Val', replaceThing(code));
```